packages/rolldown/src/rolldown-build.ts

```diff
@@ -6,11 +6,13 @@ import { createBundler } from './utils/create-bundler'
 import type { RolldownOutput } from './types/rolldown-output'
 import type { HasProperty, TypeAssert } from './utils/type-assert'
 import type { InputOptions } from './options/input-options'
+import { NormalizedInputOptions } from './options/normalized-input-options'
 
 export class RolldownBuild {
   #inputOptions: InputOptions
   #bundler?: Bundler
   #stopWorkers?: () => Promise<void>
+  normalizedInputOptions?: NormalizedInputOptions
```

Comment:
Not sure about leaking this to outside just for testing.

Reply:
that's maybe not the best way to do it, but I don't find other methods to get normalized plugin length.

Reply:
I believe testing `composeJsPlugins` directly is more proper.